src/mom5/ocean_param/vertical/ocean_vert_gotm.F90

```diff
@@ -852,7 +852,9 @@ subroutine vert_mix_gotm_bgrid (Time, Thickness, Velocity, T_prog, Dens, visc_cb
                    nuh(k)  = diff_cbt_gotm(i,j,m)
                    NN1d(k) = wrk1_v(i,j,m,1)
                    SS1d(k) = wrk1_v(i,j,m,2)
-                   dz(k)   = Thickness%dzt(i,j,m)
+                   !As Per Russ Fiedler, bug fixing:
+                   !dz(k)   = Thickness%dzt(i,j,m) 
+                   dz(k+1)   = Thickness%dzt(i,j,m) 
```

Comment:
Grid point shift

Reply:
This is a bug in the GOTM version of kpp mixing in MOM. There's a post in the mailing list by Kevin Walsh where this comes up where I identified this. I thought I had issued a pull ruquest. Looks like I didn't.